package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "eslint ./ --ext .ts",
     "lint:fix": "eslint ./ --ext .ts --fix",
     "db:migrate:create": "ts-node ./node_modules/.bin/typeorm migration:create -n",
-    "db:migrate": "ts-node ./node_modules/.bin/typeorm migration:run",
```

Comment:
What's this change for?

Reply:
Michael was running into issues on Windows 10 (and for some reason, WSL too) while running `npm run db:migrate` and `npm run db:seed` (the same error message appearing [in this issue](https://github.com/typeorm/typeorm/issues/2316)), and this change seemed to fix that while still working on my end.

I told him to try it out on WSL2 and see if that works, but yeah it might be one of those many issues with Windows 10,  not too sure.

Reply:
oof should've guessed Windows things